lib/events/dynamoevents/dynamoevents.go

```diff
@@ -333,8 +339,6 @@ func New(ctx context.Context, cfg Config) (*Log, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	otelaws.AppendMiddlewares(&awsConfig.APIOptions, otelaws.WithAttributeSetter(otelaws.DynamoDBAttributeSetter))
```

Comment:
Does `smithyoteltracing` already include dynamodb attributes? Do we care about adding them by hand, if not?